### PR TITLE
style: fixing the exampleCard in utility section.

### LINKED ISSUE #403: style: fix the exampleCard in utility section
Requirements
-The example card description should have only three lines
-Each example card should be next to the other.
-The image shold have a margin-top: 1.2rem; and margin-bottom: 1.2rem.
screen shot 2018-11-24 at 12 14 19 am

### PR DESCRIPTION
**Pull request by leyanessantiago**
fixes #403

### CODE REVIEW

File: library/styleguideComponents/ReactComponent/Utils/styles.css [hunk 1]
```diff
@@ -94,6 +94,31 @@
     color: #576574;
     align-self: flex-start;
     text-align: justify;
+    overflow: hidden;
+    max-height: 4rem;
+}
+
+.react-rainbow-utils-item_description::before {
```
Comment (TahimiLeonBravo) [inline library/styleguideComponents/ReactComponent/Utils/styles.css]: I think this is not a good solution. We can use the`text-overflow: ellipsis` style

Comment (leyanessantiago) [reply library/styleguideComponents/ReactComponent/Utils/styles.css]: The `text-overflow: ellipsis` work for a single line combined to `white-space: nowrap` but not for multiple line.

File: library/styleguideComponents/ReactComponent/Utils/styles.css [hunk 2]
```diff
@@ -94,6 +94,31 @@
...
+    content:'...';
+    position: absolute;
+    top: 10.6875rem;
+    right: 0.375rem;
```
Comment (LeandroTorresSicilia) [inline library/styleguideComponents/ReactComponent/Utils/styles.css]: with this solution you are always set the ellipsis and we need that when the description is short and fit in 3 lines then should not have ellipsis

Comment (LeandroTorresSicilia) [reply library/styleguideComponents/ReactComponent/Utils/styles.css]: sorry I saw that this works

**Pull request merged**